Test failures on 32 bits platform

On an ODROID-C1+ running Ubuntu 16.04, I'm now able to run almost all tests;

First with some prerequisites:
```
sudo apt install libusb-dev libsdl2{,-image,-mixer,-ttf,-gfx}-dev
```

Then running the tests:
```
 ~/go/src/gobot.io/x/gobot$ ./xvfb.sh go test ./...
(successes elided)
# gobot.io/x/gobot/platforms/joystick/bin
platforms/joystick/bin/scanner.go:45:50: cannot use t.Which (type sdl.JoystickID) as type int in argument to sdl.JoystickOpen
# gobot.io/x/gobot/platforms/leap
platforms/leap/leap_motion_driver_test.go:92:45: constant 4729292670 overflows int
FAIL    gobot.io/x/gobot/platforms/leap [build failed]
--- FAIL: TestMqttAdaptorConnectWithAuthError (0.00s)
        gobottest.go:14: mqtt_adaptor_test.go:93: 1 error occurred:

                * Network Error : dial tcp [::1]:1883: connect: connection refused - "*multierror.Error", should equal,  1 error occurred:

                * Network Error : Unknown protocol - "*multierror.Error"
FAIL
FAIL    gobot.io/x/gobot/platforms/mqtt 0.038s
```

- platforms/leap/leap_motion_driver_test.go seems easy to fix, the test just fails on 32 bits platform.
- platforms/joystick/bin/scanner.go is probably related to the fact I installed an old sdl2 version (2.0.4), which prints a ton of warnings. Haven't investigated yet.
- platforms/mqtt/mqtt_adaptor_test.go is kinda weird, not sure why this test exists.

Where xvfb.sh is:
```
#/bin/sh
# Run under xvfb

set -eu

if (which xvfb-run>/dev/null); then
  xvfb-run \
    -a "--server-args=-screen 0 1280x800x24 -ac -nolisten tcp -dpi 98 +extension RANDR" \
    "$@"
else
  echo "Please 'sudo apt-get install xvfb' first"
  exit 1
fi
```

--- a/platforms/leap/leap_motion_driver_test.go
+++ b/platforms/leap/leap_motion_driver_test.go
@@ -89,7 +89,7 @@ func TestLeapMotionDriverParser(t *testing.T) {
 		t.Errorf("ParseFrame incorrectly parsed frame")
 	}
 
-	gobottest.Assert(t, parsedFrame.Timestamp, 4729292670)
+	gobottest.Assert(t, parsedFrame.Timestamp, uint64(4729292670))
 	gobottest.Assert(t, parsedFrame.Hands[0].X(), 117.546)
 	gobottest.Assert(t, parsedFrame.Hands[0].Y(), 236.007)
 	gobottest.Assert(t, parsedFrame.Hands[0].Z(), 76.3394)

--- a/platforms/leap/parser.go
+++ b/platforms/leap/parser.go
@@ -66,7 +66,7 @@ type Frame struct {
 	R                [][]float64    `json:"r"`
 	S                float64        `json:"s"`
 	T                []float64      `json:"t"`
-	Timestamp        int            `json:"timestamp"`
+	Timestamp        uint64         `json:"timestamp"`
 }
 
 // X returns hand x value

